templates/category.html.tt2

```diff
@@ -23,7 +23,7 @@
 
 </head>
 
-<body onload="loadCategories()">
+<body onload="Category.loadCategories()">
```

Comment:
While adding `const` for all the remaining bundle-of-functions JS files is nice, it'd be better to switch from HTML inline events to event binding with specific IDs, like what's done on the Index and Reader: https://github.com/Difegue/LANraragi/blob/f4a62dcc1737eeb72310ed4f8f3db3d537f8cac1/public/js/index.js#L18  

Might be a bit outside of the scope of this PR for now tho.

Reply:
Yep, it's a good way to refactor the code. Actually when I was working on Categories and Upload, the coding style is far from index.js and reader.js. Don't know if it is because the time it was written or some other reasons.

Have you considered to move the inline `<script>` in `index.html.tt2` to seperated js file? So that these codes can also be checked by ESLint?

Reply:
index/reader were rewritten a few months ago and the rest of the files hasn't been updated to follow suit, yep.  
If you want a simpler example I rewrote edit.js as well: https://github.com/Difegue/LANraragi/blob/dev/public/js/edit.js

The last few scraps of inline script in `index.html.tt2` can't be moved easily, since they still pull data from the templating engine (`[% IF usingdefpass %]` and such)